src/pydicom/cli/main.py

```diff
@@ -25,12 +25,19 @@
 # Restrict the allowed syntax tightly, since use Python `eval`
 # on the expression. Do not allow callables, or assignment, for example.
 re_kywd_or_item = (
+    r"("
     r"\w+"  # Keyword (\w allows underscore, needed for file_meta)
+    r"|"  # or
+    r"\([0-9A-Fa-f]{4},[0-9A-Fa-f]{4}\)"  # DICOM hex tag (gggg,eeee)
```

Comment:
Maybe allow an optional space after the comma (was also used by the author of the issue).

Reply:
I wondered about that too - not official DICOM format, and would cause problems for people not quoting the command line, but I suppose it could be added.  Or I could scan for matches and at least produce a sensible error saying spaces not allowed in the tag.

Reply:
My vote is to not support a space

Reply:
Updated with specific error message if one or more spaces after the comma.